appinventor/components/src/com/google/appinventor/components/runtime/ListView.java

```diff
@@ -54,15 +53,15 @@
  */
 
 @DesignerComponent(version = YaVersion.LISTVIEW_COMPONENT_VERSION,
-    description = "<p>This is a visible component that displays a list of text elements." +
-        " <br> The list can be set using the ElementsFromString property" +
-        " or using the Elements block in the blocks editor. </p>",
-    category = ComponentCategory.USERINTERFACE,
-    nonVisible = false,
-    iconName = "images/listView.png")
+        description = "<p>This is a visible component that displays a list of text elements." +
```

Comment:
Reset the indentation in this section.

Reply:
I reduced the indent. Feel free to update if it's not fixed yet.